README.md

```diff
@@ -42,7 +42,7 @@ Docusaurus can manage multiple versions of your docs.
 
 Release a version v0.5.0 of your project:
 
```

Comment:
I think all of the ones mentioned here haven't neccessarily been updated in this PR, screenshot attached. Let's maybe change this to say "Control/Command+F the `/docs/` folder for the current version, and update all references to the 
<img width="378" alt="Screenshot 2022-10-20 at 13 34 35" src="https://user-images.githubusercontent.com/4981644/197030609-2ebc7d6f-943d-4e96-a2d0-361c0b5cb4a2.png">
upcoming version"

Reply:
my bad, updated the readme and the ones in verisioned_docs/version-v0.11.0/int/quickstart but didn't change the update in docs

Reply:
Okay, in future maybe the instructions aren't the most clear, but you probably shouldn't do any manual editing of the `/versioned_docs` folder. The better way to do it is:

Change only versions in the  `/docs/` folder. 

Make a git commit (optional)

Run `yarn run version v0.11.0`

That will spit out the files into versioned docs, and there should be no need to edit any of them in that folder manually.  :)